### PR TITLE
ARTEMIS-1350 - Update management console to expose client side objects

### PR DESCRIPTION
**Pull request by andytaylor**
https://issues.apache.org/jira/browse/ARTEMIS-1350

### CODE REVIEW

File: artemis-core-client/src/main/java/org/apache/activemq/artemis/api/core/management/ActiveMQServerControl.java [hunk 1]
```diff
@@ -773,6 +773,24 @@ void destroyQueue(@Parameter(name = "name", desc = "Name of the queue to destroy
...
+   boolean closeConnectionWithID(@Parameter(desc = "The connection ID", name = "ID") String ID) throws Exception;
+
+   /**
+    * Closes the session with the given id.
+    */
+   @Operation(desc = "Closes the session with the id", impact = MBeanOperationInfo.INFO)
+   boolean closeSessionWithID(String connectionID, String ID) throws Exception;
+
+   /**
+    * Closes the session with the given id.
```
Comment (michaelandrepearce) [inline artemis-core-client/src/main/java/org/apache/activemq/artemis/api/core/management/ActiveMQServerControl.java]: nit: java doc should state closes the **consumer**

File: artemis-hawtio/artemis-plugin/src/main/webapp/plugin/html/consumers.html [hunk 2]
```diff
@@ -44,9 +44,30 @@
...
+              </form>
+            </div>
         </div>
     </div>
 
+    <div hawtio-confirm-dialog="deleteDialog"
+          ok-button-text="Close"
+          on-ok="closeConsumer()">
+       <div class="dialog-body">
+         <p>You are about to close the selected session: {{gridOptions.selectedItems[0].id}}
```
Comment (michaelandrepearce) [inline artemis-hawtio/artemis-plugin/src/main/webapp/plugin/html/consumers.html]: should this be 

> You are about to close the selected **consumer**

Comment (mtaylor) [inline artemis-hawtio/artemis-plugin/src/main/webapp/plugin/html/consumers.html]: There are some issues with the layout.  The buttons do not disable when nothing select and the submission will try with empty id.  I will see if I can fix here, @andytaylor I'll send a PR to your branch.

Comment (mtaylor) [inline artemis-hawtio/artemis-plugin/src/main/webapp/plugin/html/consumers.html]: Actually, the button does seem to be disabled.  It just isn't visibly disabled, e.g. greyed out etc... I think this is OK.  I have some minor layout fixes and the comment from @michaelandrepearce .  I will squash and merge.

**Pull request closed without merging**